packages/xdl/src/TurtleConfig.ts

```diff
@@ -0,0 +1,55 @@
+import getenv from 'getenv';
```

Comment:
You don't use most of those values why copy everything?

Also, you will only need to add one more value(turtle api url), so just add this to existing config